demos/collection/jssortable.php

```diff
@@ -13,7 +13,7 @@
 
 $lister = \atk4\ui\Lister::addTo($view, [], ['List']);
 $lister->onHook('beforeRow', function ($l) {
-    $l->current_row['iso'] = strtolower($l->current_row['iso']);
+    $l->current_row['iso'] = mb_strtolower($l->current_row['iso']);
```

Comment:
why not, but I would never probably write this for ISO, which is normally ASCII only, but fine, less fixes to check year later :D

Reply:
user theoretically can input anything in iso field :D